src/schema.js

```diff
@@ -323,7 +324,7 @@ const attributes = {
 	},
 	expected_status: {
 		parents: ['beta_health', 'proxy', 'request'],
-		type: 'array',
+		type: 'tuple',
 		examples: ['sequences'],
 		arrayType: 'number'
```

Comment:
Now that the `type` value is `"tuple"`, should `arrayType` be renamed `tupleType`?

Reply:
Not crucial, as it is only used internally for the time being, whereas the type `tuple` is shown to the user in diagnostics. But yeah, we could rename that as well.